docs/java/features/odata/generate-client.md

```diff
@@ -265,8 +263,71 @@ Please note that if you use the generator for services other than SAP S/4HANA se
 
 In case you run into issues with the above process: Double check your service and file names, check that the folders are setup correctly and that the service name mappings meet your expectations.
+
+## Available Parameters
+
+The following parameters are available on the generator for both OData protocol versions:
+
+<Tabs defaultValue="maven" values={[
+{ label: 'Maven Plugin', value: 'maven', },
+{ label: 'CLI', value: 'cli', },
+{ label: 'Java', value: 'java', }]}>
+<TabItem value="maven">
+
+|   Parameter       | Default |   Description |
+|:------------------|:-------:|:--------------|
+|`<deleteOutputDirectory>`| `False` | Target directory is deleted before code generation |
+|`<defaultBasePath>`| - | Base path of the exposed API |
+|`<includeEntitySets>`| - | Only generate classes for specific entity sets |
+|`<includeFunctionImports>`| - | Only generate classes for specific function imports |
+|`<inputDirectory>`| `input` | Location of the metadata files |
+|`<namingStrategy>`| - <!--`com.sap.cloud.sdk.datamodel.odata.generator.DefaultNamingStrategy`--> | Fully-qualified Java class to be used as the naming strategy |
+|`<outputDirectory>`| `output` | Output directory for generated sources |
+|`<overwriteFiles>`| `False` | Overwrite existing files |
+|`<packageName>`| - <!-- `com.sap.cloud.sdk.s4hana.datamodel.odata` --> | Package name for the generated sources |
+|`<serviceNameMappingFile>`| - <!--`serviceNameMappings.properties`--> | Determine service names from a given mapping file |
+
+<!-- |`<nameSource>`| `LABEL` | If no custom naming strategy is used, this switches naming from label to name | -->
+
+</TabItem>
+<TabItem value="cli">
+
+|   Parameter       | Alias | Default |   Description |
+|:------------------|:-----:|:----  -:|:--------------|
+|`--default-base-path <arg>`|`-b`| - | Base path of the exposed API |
+|`--delete-output-dir`|`-d`| `False` | Target directory is deleted before code generation |
+|`--overwrite-files`|`-f`| `False` | Overwrite existing files |
+|`--input-dir <arg>`|`-i`| `input` | Location of the metadata files |
+|`--include-entity-sets <arg>`|`-ies`| - | Only generate classes for specific entity sets |
+|`--include-function-imports <arg>`|`-ifn`| - | Only generate classes for specific function imports |
+|`--name-mapping-file <arg>`|`-m`| - <!--`serviceNameMappings.properties`--> | Determine service names from a given mapping file |
+|`--name-strategy-class <arg>`|`-n`| - <!--`com.sap.cloud.sdk.datamodel.odata.generator.DefaultNamingStrategy`--> | Fully-qualified Java class to be used as the naming strategy |
+|`--output-dir <arg>`|`-o`| `output` | Output directory for generated sources |
+|`--package-name-prefix <arg>`|`-p`| - <!-- `com.sap.cloud.sdk.s4hana.datamodel.odata` --> | Package name for the generated sources |
+
+<!-- |`<nameSource>`| `LABEL` | If no custom naming strategy is used, this switches naming from label to name | -->
+
+</TabItem>
+<TabItem value="java">
+
+
```

Comment:
Add a link to latest Javadoc.
E.g., for OData v2: https://help.sap.com/doc/b579bf8578954412aea2b458e8452201/1.0/en-US/index.html?com/sap/cloud/sdk/datamodel/odata/generator/DataModelGenerator.html